### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by tomtomjhj**
Hi, thanks for nice plugin. Here are some minor improvements I found while reading the code. Please take a look at each commit.

### CODE REVIEW

File: autoload/conflict_marker/detect.vim [hunk 1]
```diff
@@ -5,19 +5,14 @@ function! conflict_marker#detect#markers() abort
...
-    finally
-        call setpos('.', pos_save)
-    endtry
+    for marker in [g:conflict_marker_begin, g:conflict_marker_separator, g:conflict_marker_end]
+        if search(marker, 'cnw') == 0
+            return 0
+        endif
+    endfor
+
+    return 1
```
Comment (rhysd) [inline autoload/conflict_marker/detect.vim]: This change does not check the order of markers. For example, the following text is detected as it contains conflict marker.

```
>>>>>>>>>>
<<<<<<<<<<
==========
```

File: plugin/conflict_marker.vim [hunk 2]
```diff
@@ -120,7 +120,7 @@ endfunction
 
 augroup ConflictMarkerDetect
     autocmd!
-    autocmd BufReadPost,FileChangedShellPost,ShellFilterPost,StdinReadPost,BufEnter,FocusGained,ColorScheme * if conflict_marker#detect#markers()
+    autocmd BufReadPost,FileChangedShellPost,ShellFilterPost,StdinReadPost * if conflict_marker#detect#markers()
```
Comment (rhysd) [inline plugin/conflict_marker.vim:123]: This will break highlighting when refreshing colorscheme (see #11)

**Review by rhysd — CHANGES_REQUESTED** [review]

File: autoload/conflict_marker/detect.vim (hunk 1 above)
Comment (tomtomjhj) [reply autoload/conflict_marker/detect.vim]: makes sense. Dropped that commit.

File: plugin/conflict_marker.vim (hunk 2 above)
Comment (tomtomjhj) [reply plugin/conflict_marker.vim:123]: As pointed out in the commit message, using `:hi-default` fixes that issue.

Comment (rhysd) [reply plugin/conflict_marker.vim:123]: Oh, I see. I didn't think that this change is linked to the above change.

**Review by rhysd — APPROVED** [review]

**Comment by rhysd** [comment]
I'll fix tests after merging this.

**Pull request merged**